packages/csv-parser-discount-code/src/main.js

```diff
@@ -47,8 +48,8 @@ export default class CsvParserDiscountCode {
   }
 
   // Remove fields with empty values from the code objects
-  static _removeEmptyFields(item: Object) {
-    return Object.keys(item).reduce((acc, key) => {
+  static _removeEmptyFields(item: Object): Object {
```

Comment:
Observation: typing with `Object` helps only to a certain degree. The more accurate we are with the type, the better we can benefit from its safety.